PowerEditor/src/ScintillaComponent/ScintillaEditView.h

```diff
@@ -469,14 +469,14 @@ friend class Finder;
 	
 
 	void setCurrentLineHiLiting(bool isHiliting, COLORREF bgColor) const {
-		execute(SCI_SETCARETLINEVISIBLE, isHiliting);
-		if (!isHiliting)
-			return;
-		execute(SCI_SETCARETLINEBACK, bgColor);
+		if (isHiliting)
+			execute(SCI_SETELEMENTCOLOUR, SC_ELEMENT_CARET_LINE_BACK, bgColor);
+		else
+			execute(SCI_RESETELEMENTCOLOUR, SC_ELEMENT_CARET_LINE_BACK, NULL);
 	};
 
 	bool isCurrentLineHiLiting() const {
-		return (execute(SCI_GETCARETLINEVISIBLE) != 0);
+		return NppParameters::getInstance().getSVP()._currentLineHilitingShow;

```

Comment:
The goal of `isCurrentLineHiLiting` is for detecting the state current line highlighting via Scintilla handle, and the same idea applies to the whole `ScintillaEditView` class.
There's no advantage to use this method since Scintilla API has been changed its behaviour. 
I suggest to remove `isCurrentLineHiLiting` method completely and use `NppParameters::getInstance().getSVP()._currentLineHilitingShow;` directly.

Reply:
Sure. Have pushed a new commit with this change.